### PR TITLE
docs: added en version of usecase_byzer

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
added en version of usecase_byzer and changed related files
fixed links and typos in zh version

### CODE REVIEW

File: docs/en/use_case/OpenMLDB_Byzer_taxi.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+### 1.1 Install OpenMLDB
+
+1. The demo will use the OpenMLDB cluster version running in Docker. See [OpenMLDB Quickstart](../quickstart/openmldb_quickstart.md) for detail installation procedures.
+2. Please modify the OpenMLDB IP configuration in order to enable the Byzer engine to access the OpenMLDB service out of the container. See [IP Configuration](../reference/ip_tips.md) for detail guidance.
+
+### 1.2 Install the Byzer Engine and the Byzer Notebook 
+
+1. For detail installation procedures of Byzer engine, see [Byzer Language Doc](https://docs.byzer.org/#/byzer-lang/en-us/).
+
+2. We have to use the [OpenMLDB plugin](https://github.com/byzer-org/byzer-extension/tree/master/byzer-openmldb) developed by Byzer to transmit messages between two platforms. To use plugin in Byzer, please configure `streaming.datalake.path`, see [the manual of Byzer Configuration](https://docs.byzer.org/#/byzer-lang/zh-cn/installation/configuration/byzer-lang-configuration) for detail.
```
Comment (michelle-qinqin) [inline docs/en/use_case/OpenMLDB_Byzer_taxi.md]: 部分Byzer文档没有对应的英文，比如这里的配置文档

Comment (lumianph) [reply docs/en/use_case/OpenMLDB_Byzer_taxi.md]: noted

File: docs/en/use_case/OpenMLDB_Byzer_taxi.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
+# Build End-to-end Machine Learning Applications Based on SQL (OpenMLDB + Byzer)
+
+This tutorial will show you how to complete a machine learning workflow with the help of [OpenMLDB](https://github.com/4paradigm/OpenMLDB) and [Byzer](https://www.byzer.org/home).
+The OpenMLDB will compute the features of the real-time data in terms of the commands and data sent by Byzer and will return the dataset been processed to Byzer for later training and prediction.
```
Comment (lumianph) [inline docs/en/use_case/OpenMLDB_Byzer_taxi.md]: OpenMLDB will compute real-time features based on the data and queries from Byzer, and then return results to Byzer for subsequent model training and inference.

File: docs/en/use_case/OpenMLDB_Byzer_taxi.md (hunk 1 above)
Comment (lumianph) [inline docs/en/use_case/OpenMLDB_Byzer_taxi.md]: To use plugin in Byzer
To use a plugin in Byzer

File: docs/en/use_case/OpenMLDB_Byzer_taxi.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+```{note}
+1. The port number of zkAddress should correspond with the files' IP configuration under the OpenMLDB `conf/` path.
+2. You can check the `streaming.plugin.clzznames` of the `\byzer.properties.override` file, which is under the `$BYZER_HOME\conf` path of Byzer, to see if the `byzer-openmldb-3.0` plugin is successfully installed. You can see the main class name `tech.mlsql.plugins.openmldb.ByzerApp` after installation. 
+3. If the plugin installation fail, download the `.jar` files and [install it offline](https://docs.byzer.org/#/byzer-lang/zh-cn/extension/installation/offline_install). 
+```
+
+### 2.3 Real-time Feature Extractions
+
+The features designed in the [OpenMLDB + LightGBM: Taxi Trip Duration Prediction](./lightgbm_demo.md) Section 2.3 will be used in this demo.
```
Comment (lumianph) [inline docs/en/use_case/OpenMLDB_Byzer_taxi.md]: designed => developed

File: docs/en/use_case/OpenMLDB_Byzer_taxi.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+from new_feature_data
+as trainning_table;
+
+```
+
+
+
+### 2.5 Training
+
+Use the `train` Byzer Lang command and its [inbuilt Linear Regression Algorithm](https://docs.byzer.org/#/byzer-lang/zh-cn/ml/algs/linear_regression) to train the model, and save it to `/model/tax-trip`.
```
Comment (lumianph) [inline docs/en/use_case/OpenMLDB_Byzer_taxi.md]: inbuilt => built-in

**Review by lumianph — CHANGES_REQUESTED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**